packages/nhsuk-frontend/src/nhsuk/components/file-upload/_index.scss

```
@@ -0,0 +1,206 @@
+@use "sass:color";
+@use "sass:math";
+@use "../../core/settings" as *;
+@use "../../core/tools" as *;
+@use "../../core/helpers" as *;
```

Probably want to copy what I've done in https://github.com/nhsuk/nhsuk-frontend/pull/1567

Making sure we forward all the nested component styles too:

```suggestion
@use "../../core/helpers" as *;
@forward "../error-message";
@forward "../hint";
@forward "../label";
```

---

Done